test/Sentry.AspNetCore.Tests/IntegrationMockedBackgroundWorker.cs

```diff
@@ -234,6 +234,7 @@ public void Environment_NotOnOptions_ValueFromEnvVar()
                 });
         }
 
+        // NOTE: Options take priority over EnvVar's.
```

Comment:
This seem wrong. I believe env var should take priority.
Good we're going to release 3.0 so we can break this behavior

Reply:
Will remove comment.

Reply:
:wrench: